config/rbac/role.yaml

```diff
@@ -15,6 +15,54 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagerconfigs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagers
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - podmonitors
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - prometheuserules
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - prometheuses
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Can we combine these to one api group?

Reply:
This is auto-generated by running `make manifests` and we should not edit that manually

Reply:
we can modify the kubebuilder marker to something like `// +kubebuilder:rbac:groups="monitoring.coreos.com",namespace=system,resources={alertmanagerconfigs,alertmanagers,prometheuses,podmonitors,servicemonitors,prometheuserules},verbs=get;list;watch` to get it generated into a single group

Reply:
Thanks, Will try that